ui/src/components/integrationFormFields/integrationFormFields.tsx

```diff
@@ -0,0 +1,68 @@
+import { IntegrationFormFieldsInterface } from 'moduleFederation/common';
+import React, { FC, useEffect } from 'react';
```

Comment:
Actually we do not need to import React here, it imports automatically in compilation time